package.json

```diff
@@ -61,7 +61,7 @@
     "karma-sauce-launcher": "^1.2.0",
     "karma-sourcemap-loader": "^0.3.7",
     "karma-webpack": "^2.0.5",
-    "mocha": "^4.0.1",
+    "mocha": "^5.0.0",
```

Comment:
Mocha 5 fails on IE 9.0.0 with the following error: https://gist.github.com/sadasant/5559112825b8f3a3e76a33364d6facd1

Reply:
Actually, it's clear [here](https://github.com/mochajs/mocha/blob/master/CHANGELOG.md#boom-breaking-changes) that Mocha dropped support for IE9 and IE10.